AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -189,6 +191,25 @@ - (void)setOpaque:(BOOL)opaque
 @synthesize delegate = _delegate;
 
 #pragma mark -
+@dynamic textView;
```

Comment:
No need for either @dynamic or @synthesize, right?  I could be wrong, but with the readonly property and implementing the getter method below, I think we should be good.  Also remove the pragma mark because it doesn't seem to be adding anything (e.g. it doesn't have a name or any real value because it is so close to the other one).